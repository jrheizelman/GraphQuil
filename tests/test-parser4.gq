/*
 *Test node creation
 */

void main(){
	int out;
	Edge b;
	
	b add ["height":6];
	out = b["height"];
}