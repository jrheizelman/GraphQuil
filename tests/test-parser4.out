Void main()
{
int_at a;
int out;
Edge b;
a = ["height": 6];
b add a;
out = b["height"];
}